tests/by-util/test_env.rs

```diff
@@ -245,3 +245,96 @@ fn test_fail_change_directory() {
         .stderr_move_str();
     assert!(out.contains("env: cannot change directory to "));
 }
+
+#[cfg(unix)]
+#[test]
+fn test_simulation_of_terminal_false() {
+    let scene = TestScenario::new(util_name!());
+
+    let out = scene.ucmd().arg("sh").arg("is_atty.sh").succeeds();
```

Comment:
Are these meaningful tests of `env`? If not, it could still be useful to have them (we're going very meta with tests for our test framework 😄), but maybe somewhere else?

Reply:
ah, yes. This is a valid point. I used `env` here as a tool rather than to test itself. This is something I should cleanup. Do you have a proposal for a better location of these tests?

Reply:
Maybe in the file where the test framework is defined? It's an interesting question, we don't really have tests for the tests yet 😄

Reply:
done